wrappers/golang/core/internal/field.go

```diff
@@ -6,30 +6,30 @@ import (
 )
 
 const (
-	BASE_LIMBS int8 = 8
+	BASE_LIMBS int = 4
```

Comment:
Limbs sum to 256 bytes.

8 * 32 = 256 
and 
4 * 64 = 256

Is this correct?